src/web/shared/LogMessages.ts

```diff
@@ -0,0 +1,31 @@
+import { Command, TaskPacket } from "./JAIAProtobuf"
+
+
+/**
+ * Messages stored in Jaia .goby and .h5 files contain a few extra fields
+ *
+ * @interface LogMessage
+ * @typedef {LogMessage}
+ */
+export interface LogMessage {
+    
+    /**
+     * Unix timestamp in microseconds since the Unix epoch
+     *
+     * @type {number}
+     */
+    _utime_: number
+
+    
+    /**
+     * Encoding scheme, where 1 is for intravehicle, non-DCCL messages 
+     * (with more precision than the equivalent DCCL messages)
+     *
+     * @type {number}
+     */
+    _scheme_: number
+}
+
+
+export type LogCommand = (LogMessage & Command)
```

Comment:
This is cool, is this what you were referring to in the interface comment that there is a sort of inheritance happening because of the few extra fields